<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>通信配置工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>681</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>节点配置</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>651</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>射频参数</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>41</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>波特率</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>75</y>
         <width>48</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>奇偶校验</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>48</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>空中速率</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>48</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>分包长度</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>40</y>
         <width>48</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>传输方式</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>75</y>
         <width>42</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>WOR周期</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>171</x>
         <y>110</y>
         <width>48</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>模块功率</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>40</y>
         <width>48</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>信道RSSI</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>70</y>
         <width>42</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>LBT使能</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>110</y>
         <width>48</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>数据RSSI</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>40</y>
         <width>48</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>模块地址</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>75</y>
         <width>48</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>频率信道</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>110</y>
         <width>24</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>密钥</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>30</y>
         <width>71</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="comboBox_6"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_7"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_8"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>30</y>
         <width>71</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QComboBox" name="comboBox_9"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_10"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_11"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>71</width>
         <height>151</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="comboBox_2"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_3"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_4"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_5"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>30</y>
         <width>61</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLineEdit" name="lineEdit_add"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_ch"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_key"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_rpa">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>150</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>读取参数</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_wpa">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>150</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>写入参数</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>651</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>操作选项</string>
      </property>
      <widget class="QTextEdit" name="textEdit_msg">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>40</y>
         <width>381</width>
         <height>81</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_clr">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>100</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>清除窗口</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_wsn">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>70</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>写入SN码</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_rsn">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>70</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>查看SN码</string>
       </property>
      </widget>
      <widget class="userComboBox" name="comboBox_com">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>40</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_com">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>40</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>打开串口</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_fac">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>100</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>恢复出厂</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>网关配置</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>651</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>端口配置</string>
      </property>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>30</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>192.168.1.1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>30</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>网关地址</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>60</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>连接网关</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>60</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>断开连接</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>391</width>
         <height>101</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>100</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>查看SN码</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>100</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>写入SN码</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>关于</string>
     </attribute>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>170</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Configuration Tool 1.0</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>userComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">usercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
